README.md

```diff
@@ -2,6 +2,12 @@
 
 The Pony toolchain multiplexer
 
+## Status
+
+[![Actions Status](https://github.com/ponylang/ponyup/workflows/vs-ponyc-latest/badge.svg)](https://github.com/ponylang/ponyup/actions)
+
+Production ready.
```

Comment:
I think this should be "beta" at most. @Theodus thoughts?

Reply:
Fair point. I've forced pushed my change.